scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/util/TestConfiguration.java

```diff
@@ -4,6 +4,6 @@
 
 @TestPropertySource(properties = { "scalingenginejob.reschedule.interval.millisecond=100",
 		"scalingenginejob.reschedule.maxcount=5", "autoscaler.scalingengine.url=http://localhost:8090",
-		"scalingengine.notification.reschedule.maxcount=2" })
+		"scalingengine.notification.reschedule.maxcount=2", "server.ssl.enabled=false" })
```

Comment:
Was not sure to proceed wrt my initial reply.So we should have unit tests SSL enabled?